src/com/facebook/buck/intellij/ideabuck/META-INF/plugin.xml

```diff
@@ -188,6 +188,36 @@
       <add-to-group anchor="last" group-id="MainToolBar"/>
     </group>
 
+    <group id="buck.test.editor">
+      <action id="buck.RunSelectedTest"
+              class="com.facebook.buck.intellij.ideabuck.actions.BuckRunSelectedTestFromProjectAction"
+              text="Run Test(s) with Buck."
+              description="Run Test(s) with Buck."
```

Comment:
Please, remove the trailing comma from test and description. Also, can you put these actions together with other actions like Run and Debug?